mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
_without-name = "%" (i don't know what to call it)_  how about "Percent"? 
 `static const char* unitSuffixes[];`in  and something like this in the corresponding cpp file:
 `unitSuffixes[] = {QT_TRANSLATE_NOOP3("unitSufixes", "px", "pixel units), QT_TRANSLATE_NOOP3("unitSuffixes", "mm", "millimeter unit"), ....};`